uninstall.ps1

```diff
@@ -0,0 +1,119 @@
+param (
+    [switch]$ForceRemove = $false,
+    [switch]$Help
+)
+
+if ($Help) {
+    Write-Host "Usage: Uninstall.ps1 [-ForceRemove] [-Help]"
+    Write-Host ""
+    Write-Host "Parameters:"
+    Write-Host "  -ForceRemove      Force removal without confirmation."
+    Write-Host "  -Help             Display this help message."
+    Write-Host ""
+    Write-Host "Examples:"
+    Write-Host "  Remove agents with confirmation:"
+    Write-Host "  .\Uninstall.ps1"
+    Write-Host ""
+    Write-Host "  Remove agents service without confirmation:"
```

Comment:
tiny nit - "Remove agents' services without confirmation:" would be more correct